### ISSUE TITLE
Formatting: Add more details from KtLint

### ISSUE THREAD

**Issue by hfhbd** [detekt/detekt#4446]
## File
```kt
fun main() {
println("Hello World!")
}
```
## Expected Behavior

```
formatting - 5min debt
	Indentation - [Unexpected indentation (0) (should be 4)] at /Users/philipwedemann/Downloads/ktlint/src/main/kotlin/Main.kt:2:1
```
## Current Behavior

```
formatting - 5min debt
	Indentation - [] at /Users/philipwedemann/Downloads/ktlint/src/main/kotlin/Main.kt:2:1
```
## Context
It would be nice, if detekt could provide more details why this line was not indented correctly.
In complex classes/cases, finding the correct indention is hard without any details.

Sample project: [SampleProject.zip](https://github.com/detekt/detekt/files/7802405/ktlint.zip)

Run `./gradlew detekt` and `./gradlew ktlintCheck` to compare its outputs.

**Comment by BraisGabin** [comment]
The reporter is working as expected. And yes, the expected behaviour is completely useless.

For example, this is what it prints when I creare a empty function named `a`

```
empty-blocks - 5min debt
        EmptyFunctionBlock - [a] at /Users/brais.gabin/Downloads/ktlint/src/main/kotlin/Main.kt:5:9
```

We need to change our default console-report from `FindingsReport` to `LiteFindingsReport`.

The output of that console-report is:

```
/Users/brais.gabin/Downloads/ktlint/src/main/kotlin/Main.kt:5:9: This empty block of code can be removed. [EmptyFunctionBlock]
/Users/brais.gabin/Downloads/ktlint/src/main/kotlin/Main.kt:2:1: Unexpected indentation (0) (should be 4) [Indentation]
```

You can enable now in your configuration, but I'm going to make it the default value for `1.20.0`

**Comment by gfreivasc** [comment]
I find that both reporters have their advantages. I like how the current default reporter lays out the issues in a way they're quick to spot on the console output (say for example on pipeline runs). However there's great value in describing the issue in cases where the rule name might not be too helpful. Does it make sense to have somewhat of a hybrid solution similar to the one suggested here?

**Comment by BraisGabin** [comment]
It is possible to create your own report of you want. I would say that we should only add a new one of the is a high demand for it. So, sure, you can open the issue and see if there is that need.
